<ui version="4.0" >
 <class>Yearly</class>
 <widget class="QDialog" name="Yearly" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>266</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>352</width>
    <height>266</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>352</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Qsalat - yearly prayer times</string>
  </property>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>120</y>
     <width>331</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title" >
    <string>Yearly generation</string>
   </property>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>Select a year</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="yearBox" >
    <property name="geometry" >
     <rect>
      <x>190</x>
      <y>30</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimum" >
     <number>1900</number>
    </property>
    <property name="maximum" >
     <number>2100</number>
    </property>
    <property name="value" >
     <number>2008</number>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="generateButton" >
   <property name="geometry" >
    <rect>
     <x>260</x>
     <y>220</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>Generate</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>30</y>
     <width>331</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title" >
    <string>Folder</string>
   </property>
   <widget class="QPushButton" name="selectButton" >
    <property name="geometry" >
     <rect>
      <x>249</x>
      <y>30</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text" >
     <string>Select</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pathEdit" >
    <property name="geometry" >
     <rect>
      <x>92</x>
      <y>30</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>Select folder</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_7" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>Generate prayer times for one year</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
